fastly/s3_test.go

```diff
@@ -711,7 +711,7 @@ func TestClient_UpdateS3_validation(t *testing.T) {
 		ServiceID:                    "foo",
 		ServiceVersion:               1,
 		Name:                         "test-service",
-		ServerSideEncryption:         S3ServerSideEncryptionKMS,
+		ServerSideEncryption:         S3ServerSideEncryptionPtr(S3ServerSideEncryptionKMS),
```

Comment:
@Integralist you'll notice that this is probably the reason most of the `const` fake enums in the SDK don't have pointers in the `PATCH` input structs! Bit of hoop jumping

Reply:
Oh I didn't see the `S3ServerSideEncryptionPtr` thing, so looks like we should replicate that approach.

Reply:
NVM just saw your commit 🙂